src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -829,6 +819,24 @@ private void Visit(ThreadFlowLocation threadFlowLocation, string threadFlowLocat
         private void Visit(Tool tool, string toolPointer)
         {
             Analyze(tool, toolPointer);
+
+            if (tool.NotificationsMetadata != null)
+            {
+                string notificationsPointer = toolPointer.AtProperty(SarifPropertyName.NotificationsMetadata);
+                for (int i = 0; i < tool.RulesMetadata.Count; ++i)
```

Comment:
>RulesMetadata [](start = 41, length = 13)

BUG! `NotificationsMetadata`. Copy/paste FTL. Apparently missing some test coverage.

Reply:
Yes! I was going to add a test but there's no prerelease v2 notifications metadata equivalent, it's entirely new to TC31. Still, we can't leave this gap: I will convert a single file to current v2 so we can close it.

---
In reply to: [255309858](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255309858) [](ancestors = 255309858)